tests/models/soil/test_carbon.py

```diff
@@ -21,10 +22,17 @@ def moist_temp_scalars(dummy_carbon_data, top_soil_layer_index):
     )
 
     moist_scalars = convert_moisture_to_scalar(
-        dummy_carbon_data["soil_moisture"][top_soil_layer_index]
+        dummy_carbon_data["soil_moisture"][top_soil_layer_index],
+        SoilConsts().moisture_scalar_coefficient,
```

Comment:
What are the brackets for? Or in other words, why `SoilConsts().bla` and not `SoilConsts.bla`?

Reply:
Turns out it doesn't do anything 😐

Reply:
I'll get rid of all instances of the brackets in tests then!